sdk/ml/azure-ai-ml/azure/ai/ml/entities/_workspace/workspace.py

```diff
@@ -240,12 +240,14 @@ def _from_rest_object(cls, rest_obj: RestWorkspace) -> "Workspace":
                 rest_obj.feature_store_settings
             )
         serverless_compute = None
-        if rest_obj.serverless_compute_settings and isinstance(
-            rest_obj.serverless_compute_settings, RestServerlessComputeSettings
-        ):
-            serverless_compute = ServerlessComputeSettings._from_rest_object(  # pylint: disable=protected-access
-                rest_obj.serverless_compute_settings
-            )
+        # TODO: Remove attribute check once serverless_compute_settings is in API response contract
+        if hasattr(rest_obj, "serverless_compute_settings"):
+            if rest_obj.serverless_compute_settings and isinstance(
```

Comment:
Can it be simplified to

if hasattr(rest_obj, "serverless_compute_settings") and isinstance(rest_obj.serverless_compute_settings, RestServerlessComputeSettings
            )

Reply:
I think this 'if hasattr(rest_obj, "serverless_compute_settings"):' should be just temporary, I'm just following the pattern here at line 219, 224

Reply:
nope, it is not temporary. It should be the correct way to check if something is not null in python.

Reply:
ok, we should check null first since it may be null even if the attribute exists.